config/stp.py

```diff
@@ -560,6 +876,7 @@ def stp_vlan_max_age(_db, vid, max_age):
     db.mod_entry('STP_VLAN', vlan_name, {'max_age': max_age})
 
 
+# config spanning_tree vlan priority <vlan-id> <0-61440>
```

Comment:
This command is valid only for PVST. Please add check.

Reply:
I have addressed this comment, please review